pymongo/collection.py

```diff
@@ -1484,21 +1484,16 @@ def find_raw_batches(self, *args, **kwargs):
           >>> for batch in cursor:
           ...     print(bson.decode_all(batch))
 
-        .. note:: find_raw_batches does not support sessions or auto
-           encryption.
+        .. note:: find_raw_batches does not support auto encryption.
```

Comment:
Do we have a ticket to implement encryption support now that we using OP_MSG?

Reply:
Yes. I just linked it to this ticket as well as the ticket that implemented OP_MSG + raw_batches.